<html><meta name="viewport" content="width=device-width, initial-scale=1.0"><head><h3>21/06/2019 02:40:03</h3></head><body><h2>cald</h2><br>  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 895, in reconnect
<br>    sock = socket.create_connection((self._host, self._port), source_address=(self._bind_address, 0))
<br>  File "/usr/lib/python2.7/socket.py", line 575, in create_connection
<br>    raise err
<br>socket.error: [Errno 111] Connection refused
<br><h2>cont</h2><br>Thu Jun 20 19:01:08 2019 contatori nessuna_risposta
<br><h2>ener</h2><br></body></html>